src/chaitanyaMankar/Array/StringOpsUsingArray.java

```diff
@@ -0,0 +1,47 @@
+/*Program 1 : (20 mins)
+Return middle character of max length String from given array.
+input : {"Krishna","Maulik","Aashvi","Technocredits","Harsh"}
+output : c*/
+
+package chaitanyaMankar.Array;
+
+public class StringOpsUsingArray 
+{
+	char MiddleChar(String Name[])
+	{	
+		String maxString = "";
+		int maxLength = 0;
+		
+		char ch = '0';
```

Comment:
always take note, we should not put 0 or any other char to assign value, instead of that you should put blank char, so whenever it comes to debug you will be clear that this char has a blank value.. if you assign some different values then you might wonder from where this 0 coming